lib/items.rb

```diff
@@ -1,20 +1,35 @@
+require 'bigdecimal'
+require 'time'
+
 class Item
-  attr_reader :id,
-              :name,
-              :description,
-              :unit_price,
-              :created_at,
-              :updated_at,
-              :merchant_id,
-              :unit_price_to_dollars
+  attr_accessor :id,
+                :name,
+                :description,
+                :unit_price,
+                :created_at,
+                :updated_at,
+                :merchant_id
 
   def initialize(attributes)
-    @id = attributes[:id]
+    @id = attributes[:id].to_i
     @name = attributes[:name]
     @description = attributes[:description]
-    @unit_price = attributes[:unit_price]
+    @unit_price = (BigDecimal(attributes[:unit_price])/100)
```

Comment:
I think we can actually remove the parentheses here, It might look a bit cleaner.